core/src/main/java/org/apache/brooklyn/feed/ssh/SshFeed.java

```diff
@@ -28,6 +28,19 @@
 import java.util.concurrent.Callable;
 import java.util.concurrent.TimeUnit;
 
+import org.slf4j.Logger;
```

Comment:
Would prefer us not reordering imports (most of our code is in the default order that eclipse gives - but this change is not). But I can live with it.